de-lib/src/main/java/org/iplantc/de/fileViewers/client/presenter/FileViewerPresenterImpl.java

```diff
@@ -484,19 +483,6 @@ void setFileViewerDebugIds() {
         viewers.forEach(viewer -> viewer.asWidget().ensureDebugId(baseID));
     }
 
-    /**
-     * Calls the tree URL service to fetch the URLs to display in the grid.
-     */
-    void callTreeCreateService(final FileViewer viewer, File file) {
-        simpleContainer.mask(appearance.retrieveTreeUrlsMask());
-        IsMaskable maskable = asMaskable(simpleContainer);
-        fileEditorService.getTreeUrl(file.getPath(),
```

Comment:
The `getTreeUrl` method can probably also be removed from the interface `FileEditorServiceFacade`